examples/widgets/image/lv_example_image_1.c

```diff
@@ -7,10 +7,15 @@ void lv_example_image_1(void)
     lv_obj_t * img1 = lv_image_create(lv_screen_active());
     lv_image_set_src(img1, &img_cogwheel_argb);
     lv_obj_align(img1, LV_ALIGN_CENTER, 0, 0);
+    lv_image_set_scale_x(img1, 512);
+    lv_image_set_scale_y(img1, 128);
+    lv_image_set_rotation(img1, 10);
```

Comment:
Is this change intended? `example_image_1` is the start point to use image, better leave it as simple as possible.